packages/audit/scripts/log-viewer.py

```
@@ -0,0 +1,111 @@
+        summary = data.get("summary") or {}
+        detail = f"events={data.get('event_count', 0)} errors={summary.get('errors', 0)}"
+
+    return f"[{ts}] sid={sid} tid={tid} {event_type:16s} {detail}"
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description="Audit log viewer for ai-orchestra")
+    parser.add_argument("--type", dest="event_type", help="イベントタイプでフィルタ")
+    parser.add_argument("--session", help="セッション ID でフィルタ")
+    parser.add_argument("--trace", help="トレース ID でチェーン追跡")
+    parser.add_argument("--limit", type=int, default=100, help="表示件数（デフォルト: 100）")
```

Ruff (0.15.9)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)